kroxylicious-operator/src/test/resources/DerivedResourcesTest/unsupported-cluster-tls-certificate/cond-Accepted-one.yaml

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+---
+cluster: "one"
+type: "Accepted"
+status: "False"
+reason: "Invalid"
+message: "Resource in group \"\" of kind \"ThisKindDoesNotExist\" with name \"kafka-client-secret\"\
+  \ referenced from  spec.targetCluster.tls.certificateRef is not supported. Supported\
+  \ kinds and groups: kind: \"Secret\" in group \"\""
```

Comment:
I think it would be a good idea if we had a consistent/uniform way of reporting references which are not supported (i.e. bad kind/group) or are not found. 

* For "not supported" we could use CEL
* But for "not found" the operator has to do it, and there are lots of places we're using references, so I think including the path within the CR is going to be helpful to the user.

Reply:
We now use a single `ClusterCondition` for not found. While that will get refactored, since it's not really just a condition we expect to use on `VirtualKafkaCluster`, we got to a single method.